garak/resources/tap/tap_main.py

```diff
@@ -34,7 +35,7 @@
 SAVE_RESULTS = True
 
 resources_tap_data_file = (
-    garak._config.transient.basedir
+    garak._config.transient.cache_dir
```

Comment:
TAP has a pre-populated file in resources/tap/data that is distributed with garak. Maybe is makes sense to separate pre-loaded TAP resources from resources created while running TAP. The latter might belong in XDG data.

Reply:
Same reasoning [here](https://github.com/leondz/garak/pull/799/files/bdc2a416078522b9dd0052b5036e834f27598a21#r1695258578). Although this comment does expose that there is likely an enhancement to be done here as the `cached` version is only consumed in a live run an not in `TAPCached` by default.

This again relates to this [idea](https://github.com/leondz/garak/pull/799/files/bdc2a416078522b9dd0052b5036e834f27598a21#r1693172648).